e2e/runner/staking.go

```diff
@@ -0,0 +1,112 @@
+package runner
+
+import (
+	"fmt"
+
+	sdkmath "cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	observertypes "github.com/zeta-chain/node/x/observer/types"
+)
+
+// UnStakeToBelowMinimumObserverDelegation  unstakes the validator self delegation to below the minimum observer delegation
```

Comment:
```suggestion
// UnstakeToBelowMinimumObserverDelegation unstakes the validator self delegation to below the minimum observer delegation
```
Please check sentence structure + unstake is a single word

Reply:
Refactored to use undelegate terminology as it's more inline with the cosmos api 
3a7fb5051294aec5a4ccc4f01364f46a65b0bca8